meinberlin/assets/scss/components/_dropdown.scss

```diff
@@ -42,7 +42,7 @@
     max-width: none;
     padding: 0;
     margin: 0;
-    z-index: 2;
+    z-index: 3;
```

Comment:
@phillimorland I was not allowed to comment on the lines above, so here: At the top of this file (line 16-19), there is another z-index, that might be there to be higher than the one here?! Maybe instead of uping this one, we can remove the other one?

Reply:
Hmm: https://github.com/liqd/a4-meinberlin/commit/10d2ca628cdf365c4eae6f31aa6aef6799c39aad I don't know, why it's there, maybe it can go?

Reply:
this makes the boxshadow overlap the toggle, hence the z-index on line 19, making them the same also causes overlap so I would avoid if possible. 
i'm not sure if this fully solves the issue either. I don't think hiding the dropdown toggle under the dropdown is great for UX and may cause further issues if the list is very long. We would be better trying to make it drop down and extend beyond the bottom of the map, what do you think?